source/include/jobs.h

```diff
@@ -314,6 +317,32 @@ typedef enum
     JobsMaxTopic
 } JobsTopic_t;
 
+/**
+ * @ingroup jobs_struct_types
+ * @brief Structure for Jobs_UpdateMsg request parameters.
+ *
+ * @note For optional fields setting a pointer to NULL or
+ * the length to 0U will disable this field from being used
+ *
+ *
+ * @note Optional fields include:
+ *    * expectedVersion
+ *    * expectedVersionLength
+ *    * statusDetails
+ *    * statusDetailsLength
+ *
+ * @note The status details must be a JSON formatted key-value
+ * paid.
+ */
+typedef struct
+{
+    JobCurrentStatus_t status;    /**< Status to update the job to */
```

Comment:
NIT: can we add period '.' to end of comments/sentences?

Reply:
It seems that there are a lot of places where this is missing - feel free to ignore.

Reply:
I've double checked my comments and will add periods for consistency.